### PR TITLE
Adding split arc automation changes tests/config_control_port_with_li…

### PR DESCRIPTION
**Pull request by depasupu**
…nk.py

Signed-off-by: Devaki Pasupuleti <devaki.pasupuleti@intel.com>

### CODE REVIEW

**Comment by dkaranjai** [comment]
There is a mismatch in the number of file between this PR and PR #8

**Comment by depasupu** [comment]
Reattaching new logs

[dpdk_action_selector_cli.txt](https://github.com/ipdk-io/ptf-tests/files/10290327/dpdk_action_selector_cli.txt)
[dpdk_hotplug_remove.txt](https://github.com/ipdk-io/ptf-tests/files/10290328/dpdk_hotplug_remove.txt)
[dpdk_hotplug_remove_Multi_port.txt](https://github.com/ipdk-io/ptf-tests/files/10290329/dpdk_hotplug_remove_Multi_port.txt)
[dpdk_l3em_counter_reset.txt](https://github.com/ipdk-io/ptf-tests/files/10290330/dpdk_l3em_counter_reset.txt)
[l2_exact_match_with_link_port.txt](https://github.com/ipdk-io/ptf-tests/files/10290331/l2_exact_match_with_link_port.txt)
[config_control_port_with_link.txt](https://github.com/ipdk-io/ptf-tests/files/10290332/config_control_port_with_link.txt)
[dpdk_action_profile_cli.txt](https://github.com/ipdk-io/ptf-tests/files/10290334/dpdk_action_profile_cli.txt)
[DPDK_hotplug_Add_Remove_multiple_times.txt](https://github.com/ipdk-io/ptf-tests/files/10291038/DPDK_hotplug_Add_Remove_multiple_times.txt)

File: tests/dpdk_l3em_indirect_counter.py [hunk 1]
```diff
@@ -14,9 +14,10 @@
 # limitations under the License.
 
 """
-DPDK L3EM Indirect counter
+DPDK L3EM reset Indirect counter
```
Comment (dkaranjai) [inline tests/dpdk_l3em_indirect_counter.py]: why rename?

File: tests/dpdk_l3em_indirect_counter.py [hunk 2]
```diff
@@ -29,229 +30,176 @@
...
-from common.utils.config_file_utils import get_config_dict, get_gnmi_params_simple
-from common.utils.gnmi_ctl_utils import gnmi_ctl_set_and_verify
 from common.lib.telnet_connection import connectionManager
+from common.utils.config_file_utils import (
+    get_config_dict,
+    get_gnmi_params_simple,
+)
+from common.utils.gnmi_ctl_utils import gnmi_ctl_set_and_verify, gnmi_set_params, ip_set_ipv4
 
+class IPDK_L3EM_INDIRECT_COUNTER_RESET(BaseTest):
```
Comment (dkaranjai) [inline tests/dpdk_l3em_indirect_counter.py]: reset is a separate test case

**Review by dkaranjai — CHANGES_REQUESTED** [review]

**Review by hirentha — COMMENTED** [review]
Still print is every where, please replace print with log.info [where info is provide.], log.passed[ where pass statement is there] and log.failed[ where fail statement is there ]

File: tests/dpdk_l3em_indirect_counter.py (hunk 2 above)
Comment (depasupu) [reply tests/dpdk_l3em_indirect_counter.py]: adding dpdk_l3em_indirect_counter_reset.py

**Comment by depasupu** [comment]
> Still print is every where, please replace print with log.info [where info is provide.], log.passed[ where pass statement is there] and log.failed[ where fail statement is there ]

==
Changed to log.info/log.passed as per statement

**Review by dkaranjai — APPROVED** [review]

**Pull request merged**